shell.nix

```diff
@@ -5,6 +5,6 @@ in
   haskell.lib.buildStackProject {
      ghc = haskell.packages.${lts}.ghc;
      name = "rscoin";
-     buildInputs = [ zlib glib cairo gnome.pango gnome3.gtk git cabal-install
+     buildInputs = [ zlib.out zlib.dev glib cairo gnome.pango gnome3.gtk git cabal-install
```

Comment:
@domenkozar yes, i tried many times to make it work without `shell.nix`. And i also tried this particular approach. Have no idea why does it work now. If you can confirm it's working, it'd be nice to have it in this pr too.